README.rst

```diff
@@ -0,0 +1,156 @@
+DIRACOS
+=======
+
+|pipeline status|
+
+DIRACOS aims at bringing in one archive all the dependencies needed by
+DIRAC to run Agents, Services and clients. It is not intended to perform
+interactive tasks (debugging, file editing, etc). We are aware than some
+“basic” tools won’t work (less, emacs, etc). They won’t be fixed.
+
+-  `Principle <docs/0_concepts.rst#principle>`__
```

Comment:
It can go like this, but then we will be able to remove this hand made summary and use the RST generated one :-)

Reply:
If we do that it will no longer appear on GitHub: https://github.com/github/markup/issues/534

Reply:
ah :-(